class-tgm-plugin-activation.php

```diff
@@ -1519,7 +1519,9 @@ protected function get_plugins_api( $slug ) {
 
 				if ( is_wp_error( $response ) ) {
 					if ( true === WP_DEBUG ) {
+						// @codingStandardsIgnoreStart
 						wp_die( esc_html( $this->strings['oops'] ) . var_dump( $api ) ); // WPCS: xss ok.
+						// @codingStandardsIgnoreEnd
 					} else {
```

Comment:
This will ignore that line always. As it only generates a warning and it's a valid warning which we may need to revisit, ignore warnings in travis and let this show up in manual/local runs.